.babelrc

```diff
@@ -1,4 +0,0 @@
-{
-  "presets": ["next/babel"],
-  "plugins": ["@vanilla-extract/babel-plugin"]
-}
```

Comment:
`.babelrc`消したってことはNext.jsがSWCで起動するようになった？

Reply:
多分そういうことですね！next/fontが動かせる環境が整った！

Reply:
開発サーバの起動時になんかそういう情報出た気がするからログ貼って欲しいかもー

Reply:
ぱっと見なにも出てないんすよね。。。（出てなければOKなんかな🤔）

```bash
$ yarn dev
yarn run v1.22.19
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /Users/takashi/the_work/uyupun/official/.env
event - compiled client and server successfully in 7.1s (177 modules)
wait  - compiling / (client and server)...
event - compiled client and server successfully in 111 ms (180 modules)
```

Reply:
babel入ってるときの起動ログ

```
$ yarn dev
yarn run v1.22.19
$ next dev
ready - started server on 0.0.0.0:3001, url: http://localhost:3001
info  - Loaded env from /Users/k-tobita/Desktop/index/workspace/official/.env
warn  - Port 3000 is in use, trying 3001 instead.
info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
info  - Using external babel configuration from /Users/k-tobita/Desktop/index/workspace/official/.babelrc
event - compiled client and server successfully in 20.5s (206 modules)
wait  - compiling / (client and server)...
event - compiled client and server successfully in 3.6s (209 modules)
```

Reply:
```
info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
```

がなくなってるからOKなんだろうか

Reply:
多分。。。